resources/github-actions.md

```diff
@@ -45,6 +45,61 @@ There is a link provided at the end of every C++ and Python build job where the
 
 ![](/assets/images/downloads.png)
 
+## Using CI Artifacts in Other PRs
```

Comment:
```suggestion
## Using Conda CI Artifacts in Other PRs
```

Should we clarify this section header since it only applies to Conda CI artifacts?